doc/classes/Transform3D.xml

```diff
@@ -235,35 +255,35 @@
 			<return type="Transform3D" />
 			<param index="0" name="right" type="float" />
 			<description>
-				This operator multiplies all components of the [Transform3D], including the [member origin] vector, which scales it uniformly.
+				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale proportionately.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Transform3D" />
 			<param index="0" name="right" type="int" />
 			<description>
-				This operator multiplies all components of the [Transform3D], including the [member origin] vector, which scales it uniformly.
+				Multiplies all components of the [Transform3D] by the given [int], including the [member origin]. This affects the transform's scale proportionately.
 			</description>
 		</operator>
 		<operator name="operator /">
 			<return type="Transform3D" />
 			<param index="0" name="right" type="float" />
 			<description>
-				This operator divides all components of the [Transform3D], including the [member origin] vector, which inversely scales it uniformly.
+				Divides all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale proportionately.
 			</description>
 		</operator>
 		<operator name="operator /">
 			<return type="Transform3D" />
 			<param index="0" name="right" type="int" />
 			<description>
-				This operator divides all components of the [Transform3D], including the [member origin] vector, which inversely scales it uniformly.
+				Divides all components of the [Transform3D] by the given [int], including the [member origin]. This affects the transform's scale proportionately.
 			</description>
 		</operator>
 		<operator name="operator ==">
 			<return type="bool" />
 			<param index="0" name="right" type="Transform3D" />
 			<description>
-				Returns [code]true[/code] if the transforms are exactly equal.
+				Returns [code]true[/code] if the components of both transforms are exactly equal.
```

Comment:
```suggestion
				Returns [code]true[/code] if each component of both transforms are exactly equal.
```
Might read better, just a suggestion

Reply:
Decided against it, feel free to contest it